Source/ARTWebSocketTransport.m

```diff
@@ -184,7 +184,7 @@ - (NSURL *)setupWebSocket:(NSDictionary<NSString *, NSURLQueryItem *> *)params w
     [queryItems addValueAsURLQueryItem:[ARTDefault version] forKey:@"v"];
     
     // Lib
-    [queryItems addValueAsURLQueryItem:[ARTDefault agent] forKey:@"lib"];
+    [queryItems addValueAsURLQueryItem:[options agents] forKey:@"lib"];
```

Comment:
Not sure if it still should be 'lib', in android it's 'agent'. @QuintinWillison

Reply:
Well spotted. I think you're probably right. But please go back to [the spec](https://docs.ably.com/client-lib-development-guide/features/#RSC7d3) and double check.

Reply:
It's `agent`

Reply:
https://github.com/ably/ably-cocoa/pull/1119/commits/a09039f4fd081eb71dc408d6c5d52fe1e9ffcae1